pkgs/development/compilers/gforth/default.nix

```diff
@@ -26,15 +27,17 @@ in stdenv.mkDerivation rec {
 
   passthru = { inherit bootForth; };
 
-  configureFlags = lib.optional stdenv.isDarwin [ "--build=x86_64-apple-darwin" ];
+  configureFlags = [
+    "--with-lispdir=${lispDir}"
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
&& stdenv.isx86_64?

Reply:
:eyes: That's a good point. May as well fix while I'm here.